packages/notifi-react/lib/components/NavHeader.tsx

```diff
@@ -16,6 +17,8 @@ export type NavHeaderProps = {
 export const NavHeader: React.FC<PropsWithChildren<NavHeaderProps>> = (
   props,
 ) => {
+  const { globalCtas } = useGlobalStateContext();
```

Comment:
Rather than hardcode the globalCtas in this optional action component, we can pass the action related props in `NavHeader.tsx`
Then we can add add the respective props in the component which consume the `NaveHeader.tsx`. For example in `FtuAlertEdit.tsx`, we can do like below:
```ts
export type FtuAlertEditProps = {
  // ...others
  navHeaderRightCta?: {
    action: string; // We need to have a respective action in globalCtas,
    icon: IconType;
  };
};

// ...others

export const FtuAlertEdit: React.FC<FtuAlertEditProps> = (props) => {
  const { updateFtuStage } = useNotifiUserSettingContext();
  const [isLoading, setIsLoading] = React.useState(false);

  const onClick = async () => {
    setIsLoading(true);
    await updateFtuStage(FtuStage.Done);
    setIsLoading(false);
    props.onClickNext();
  };

  const { globalCtas } = useGlobalStateContext();

  return (
      // ...others
      <NavHeader
        // ...others
        rightCta={
          props.navHeaderRightCta &&
          globalCtas?.[props.navHeaderRightCta.action]
            ? {
                icon: props.navHeaderRightCta.icon,
                action: () => globalCtas[props.navHeaderRightCta!.action](),
              }
            : undefined
        }
      >
      // ... others
  )
};
```
IMO, This approach is preferable because we can grant more flexability to users (developers) so they will be able to do whatever action for the navBar topRightCta instead of being limited as `onClose`